source/features/table-input.tsx

```diff
@@ -0,0 +1,79 @@
+import './table-input.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import TableIcon from 'octicon/diff-added.svg';
+import * as pageDetect from 'github-url-detection';
+import * as textFieldEdit from 'text-field-edit';
+
+import features from '.';
+
+function generateGfmTable([w, h]: [number, number]): string {
+	return `${'|     '.repeat(w)}|\n${'|-----'.repeat(w)}|\n${`${'|     '.repeat(w)}|\n`.repeat(h)}`.slice(0, -1);
+}
+
+function addTable(event: delegate.Event<MouseEvent, HTMLButtonElement>): void {
+	event.preventDefault();
+	(event.delegateTarget.parentElement!.parentElement as HTMLDetailsElement)!.open = false;
+
+	const field = event.delegateTarget.form!.querySelector('textarea')!;
+	const endsWithNl = field.value.length && !field.value.endsWith('\n');
+	const cursorPos = field.value.length + (endsWithNl ? 4 : 3);
```

Comment:
```suggestion
	const cursorPosition = field.value.length + (endsWithNl ? 4 : 3);
```

Don't use abbreviations. Applies to all identifiers.